Tessian/Tessian.json

```diff
@@ -0,0 +1,1395 @@
+                            "queryKey": "A",
+                            "metricsQueryMode": null,
+                            "metricsQueryData": null,
+                            "tracesQueryData": null,
+                            "parseMode": "Auto",
+                            "timeSource": "Message"
+                        }
+                    ],
+                    "description": "",
+                    "timeRange": null,
+                    "coloringRules": null,
+                    "linkedDashboards": []
+                }
+            ],
+            "variables": [],
+            "coloringRules": []
+        },
+        {
+            "type": "DashboardV2SyncDefinition",
+            "name": "Tessian - Enforcer",
+            "description": "View stats and metrics related to Tessian Enforcer.",
+            "title": "Tessian - Enforcer",
+            "rootPanel": null,
+            "theme": "Dark",
+            "topologyLabelMap": {
+                "data": {}
+            },
+            "refreshInterval": 0,
+            "timeRange": {
+                "type": "BeginBoundedTimeRange",
+                "from": {
+                    "type": "RelativeTimeRangeBoundary",
+                    "relativeTime": "-1w"
+                },
+                "to": null
+            },
+            "layout": {
+                "layoutType": "Grid",
+                "layoutStructures": [
+                    {
+                        "key": "panelPANE-0A4FC934B407984D",
+                        "structure": "{\"height\":8,\"width\":14,\"x\":0,\"y\":8}"
+                    },
+                    {
+                        "key": "panelPANE-3E9BF498909E9A40",
+                        "structure": "{\"height\":7,\"width\":7,\"x\":0,\"y\":0}"
+                    },
+                    {
+                        "key": "panelPANE-192F1BD6A13D9B4E",
+                        "structure": "{\"height\":10,\"width\":7,\"x\":0,\"y\":16}"
+                    },
+                    {
+                        "key": "panelPANE-7D524B71AE11F943",
+                        "structure": "{\"height\":10,\"width\":7,\"x\":7,\"y\":16}"
+                    },
+                    {
+                        "key": "panelPANE-49112418BBC59A4C",
+                        "structure": "{\"height\":25,\"width\":4,\"x\":14,\"y\":0}"
+                    },
+                    {
+                        "key": "panelFA5D0278A5E4C844",
+                        "structure": "{\"height\":25,\"width\":4,\"x\":18,\"y\":0}"
+                    },
+                    {
+                        "key": "panelC7FAA16DAED08A4A",
+                        "structure": "{\"height\":7,\"width\":7,\"x\":7,\"y\":3}"
+                    }
+                ]
+            },
+            "panels": [
+                {
+                    "id": null,
+                    "key": "panelPANE-0A4FC934B407984D",
+                    "title": "Unauthorized Email Attempts",
```

Comment:
The pie chart and trend chart have same title but the data count is different. Seems they both show different info. Is this expected? or do we need to update the title?
![image](https://user-images.githubusercontent.com/3620468/128375508-9ae764f0-6732-43dc-b9f6-1b62d22274b0.png)

Reply:
when refactoring the query I removed the `where module 'enforcer'` by mistake. It should be fixed now